lib/tasks/populate.rake

```diff
@@ -326,8 +326,6 @@ namespace :spree_roles do
         build_permission_group(
           [
             [:admin], Spree::Store,
-            [:admin, :manage], Spree::StoreCreditCategory,
```

Comment:
I think we should create two separate branches one for 3-0-stable and another for 3-1-stable and above.